tutorials/nservicebus-101/lesson-1/tutorial.md

```diff
@@ -0,0 +1,144 @@
+
+
+#### Error queue
+
+snippet:ErrorQueue
+
+Sometimes processing a message fails due to an unanticipated input, a coding bug, or even something as simple as a database deadlock. Automatic retries will make dealing with deadlocks a non-issue, but for very serious errors, the message could get stuck at the top of the queue and be retried indefinitely. This type of message, known as a **poison message**, would block all other messages behind it. When these occur, NServiceBus needs to be able to set it aside in a different queue to allow other work to get done. This queue is referred to as the **error queue** and is commonly named `error`. We will discuss [**recoverability**](/nservicebus/recoverability/) more in [Lesson 5: Retrying errors](../lesson-5/).
+
+
+#### Installers
+
+snippet:EnableInstallers
+
+This setting instructs the NServiceBus endpoint to run its installers on startup. Installers are used to set up anything the endpoint requires to run. The most common example is creating necessary queues, such as the endpoint's input queue where it will receive messages.
+
+
+### Starting up
+
+At the end of the `AsyncMain` method, after the configuration code, add the following code which will start up the endpoint, keep it running until we press the Enter key, and then shut it down.
+
+snippet:Startup
+
+When the endpoint starts, the `EndpointConfiguration` information is locked down from further changes, and the settings it contains are used to initialize the endpoint.
+
+When you run the endpoint for the first time, the endpoint will:
+
+ * Display its logging information, which is written to a file as well as the console, so if you miss something on the console, you can go back and look at the file. NServiceBus also logs to multiple levels, so you can [change the log level](/nservicebus/logging/) from `INFO` to log level `DEBUG` in order to get more information about what's going on.
+ * Display the [status of your license](/nservicebus/licensing/).
+ * Attempt to add the current user to the "Performance Monitor Users" group so that it can write [performance counters](/nservicebus/operations/performance-counters.md) to track its health and progress. If it is not able to do so (usually because the process is not running with elevated privileges) then it will show you how to do it manually from an admin console.
+ * Warn you that the [queues it created have too many permissions](/nservicebus/msmq/operations-scripting.md#create-queues-default-permissions). This makes it easier for development, but in a production scenario these queues should be created with the minimum required privileges, and these warnings will serve as reminders to do that.
```

Comment:
Current log message is:

> INFO  NServiceBus.Features.QueuePermissions Queue [private$\error] is running with [Everyone] and [NT AUTHORITY\ANONYMOUS LOGON] permissions. Consider setting appropriate permissions, if required by the organization. For more information, consult the documentation.

I'm thinking now something like "development-specific permissions that may not be required in production" ?